hailolibero/hailolibero.py

```diff
@@ -4,17 +4,50 @@
 import os
 from aiohttp import *
 from furl import furl
+from bs4 import BeautifulSoup
 
+import typing
+from pydantic import BaseModel
+
+class HailoValue(BaseModel):
+    value: int
+    min: int
+    max: int
+
+class HailoSettings(BaseModel):
+
+    led: HailoValue
+    pwr: HailoValue
+    dist: HailoValue
+
+    ssid: str
+    key: str
+
+    # dhcp = 1, static = 0
+    ipconf: bool
+
+    ip: str = None
+    subnet: str = None
+    gateway: str  = None
+
+class HailoInfo(BaseModel):
+
+    device: str
+    firmware: str
+
+    status: str
```

Comment:
We should enum this in the future, I don't know all the possible states currently which I guess you also didn't :)